lib/tasks/pubmed.rake

```diff
@@ -44,79 +49,73 @@ namespace :pubmed do
           begin
             result = pub.update_from_pubmed
             pubs_updated_count += 1 if result
-          rescue  => e
+          rescue => e
             message = "*****ERROR on cap_profile_id #{id} for publication_id #{pub.id}: #{e.message}"
             puts message
             logger.error message
             error_count += 1
           end
         end
         authors_found += 1
-      else
-        authors_not_found += 1
-        message = "*****cap_profile_id #{id} not found"
-        puts message
-        logger.error message
-      end
-      if error_count > max_errors
-        message = "Halting: Maximum number of errors #{max_errors} reached"
-        logger.error message
-        raise message
       end
+      next unless error_count > max_errors
```

Comment:
I don't really care about this, but I find it easier to read: `next if error_count < max_errors`

Reply:
This isn't my code and wasn't flagged by rubocop.